target-app/Networking/Services/TargetServices.swift

```diff
@@ -43,4 +43,23 @@ class TargetServices {
         }
     }
     
+    class func get(
+        completion: @escaping (Result<[TargetElement], Error>) -> Void
+    ) {
+        BaseAPIClient.default.request(
+            endpoint: TargetEndpoint.get
+        ) { (result: Result<GetTargetResponse?, Error>, responseHeaders) in
+            switch result {
+            case .success(let getTargetResponse):
+                if let targets = getTargetResponse?.targets {
+                    completion(.success(targets))
+                } else {
+                    completion(.failure(AuthError.userSessionInvalid))
```

Comment:
This is not true probably. Maybe it is a good idea to define custom errors for when the targets can't be parsed.

Reply:
done!